lib/modules/keyboardNav.js

```diff
@@ -1078,16 +1073,29 @@ module.toggleChildren = function() {
 	click(thisToggle);
 };
 
-module.toggleExpando = function() {
-	const selected = SelectedEntry.selectedThing();
-	if (!selected) return;
+module.toggleExpandos = function(scrollOnExpanding) {
```

Comment:
I think this could be a local function instead of a property on `module`; I probably missed it while converting the modules